### PR TITLE
(WIP) Build status icon overhaul

### PR DESCRIPTION
**Pull request by zdhickman**
- [x] Fix colorblind issue (overlay icon on colored circle)
- [x] All icons consistent
- [x] Add skipped icon
- [x] Fix in-progress icons
- [x] Improve styling on icons (size, position on circle, and other small details)
- [x] Code clean-up all over the place

For another time?:
- [ ] make blinking icon work on starred dashboard page (need to have more data passed down)

Example page (give your thoughts on the appearance):

screen shot 2016-02-05 at 5 24 11 pm

### CODE REVIEW

**Comment by benrlodge** [comment]
:fancy:

File: BlazarUI/app/stylus/components/icon.styl [hunk 1]
```diff
@@ -11,3 +11,14 @@
 
 .fa-link
   color $link-color
+
+.fa-stack-2x
+  font-size 1.5em
+  
+.fa-stack-1x
+  font-size .75em
+  line-height 2em
```
Comment (benrlodge) [inline BlazarUI/app/stylus/components/icon.styl]: i would stick with unit-less line heights.

File: BlazarUI/app/scripts/components/shared/IconStack.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+class IconStack extends Component {
+
+  getBaseClassNames() {
+    return ClassNames([
+      "fa-stack-2x",
+      this.props.classNames
+    ]);
+  }
+	
+	renderIconStackBase() {
```
Comment (benrlodge) [inline BlazarUI/app/scripts/components/shared/IconStack.jsx]: looks like some wind blew through this file

**Comment by benrlodge** [comment]
:+1: looking good so far

**Comment by zdhickman** [comment]
I'm ready to merge this PR as far as features go. If anyone wants to give this another run-through, feel free

File: BlazarUI/app/scripts/components/shared/IconStack.jsx (hunk 2 above)
Comment (benrlodge) [inline BlazarUI/app/scripts/components/shared/IconStack.jsx]: has the storm calmed down?

Comment (zdhickman) [inline BlazarUI/app/scripts/components/shared/IconStack.jsx]: :lol: what, you don't like 4-character tabs?

**Pull request merged**